src/query/storage/converter_test.go

```diff
@@ -229,3 +230,45 @@ func TestPromReadQueryToM3(t *testing.T) {
 		})
 	}
 }
+
+var (
+	benchResult *prompb.QueryResult
+)
+
+func BenchmarkFetchResultToPromResult(b *testing.B) {
```

Comment:
Might be useful to paste the benchmark results in a comment here just for easier comparisons in the future?